src/App.scss

```diff
@@ -21,16 +21,16 @@ iframe {
   &__header {
     margin-bottom: 1em;
   }
-}
+} */
 
-.UserInfo {
+/* .UserInfo {
```

Comment:
The CSS styling for the .UserInfo component has been commented out which might affect the display of the user information.